src/main/java/org/fundacionjala/coding/ariel/Kata1.java

```diff
@@ -0,0 +1,25 @@
+package org.fundacionjala.coding.ariel;
+
+
+/**
+ * Kata 1.
+ * Ariel Gonzales.
+ */
+public class Kata1 {
+
+    public static final int CONDITION = 5;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AIndentationCheck)